app/Command/Compile.hs

```diff
@@ -41,25 +41,25 @@ data PSCMakeOptions = PSCMakeOptions
 printWarningsAndErrors :: Bool -> Bool -> P.MultipleErrors -> Either P.MultipleErrors a -> IO ()
 printWarningsAndErrors verbose False warnings errors = do
   pwd <- getCurrentDirectory
-  cc <- bool Nothing (Just P.defaultCodeColor) <$> ANSI.hSupportsANSI stderr
+  cc <- bool Nothing (Just P.defaultCodeColor) <$> ANSI.hSupportsANSI stdout
   let ppeOpts = P.defaultPPEOptions { P.ppeCodeColor = cc, P.ppeFull = verbose, P.ppeRelativeDirectory = pwd }
   when (P.nonEmpty warnings) $
     hPutStrLn stderr (P.prettyPrintMultipleWarnings ppeOpts warnings)
   case errors of
     Left errs -> do
-      hPutStrLn stderr (P.prettyPrintMultipleErrors ppeOpts errs)
+      hPutStrLn stdout (P.prettyPrintMultipleErrors ppeOpts errs)
       exitFailure
```

Comment:
Was your earlier comment about swapping it to `stdout` also apply on line 47:
```haskell
-- current
hPutStrLn stderr (P.prettyPrintMultipleWarnings ppeOpts warnings)

-- desired?
hPutStrLn stdout (P.prettyPrintMultipleWarnings ppeOpts warnings)
```

Reply:
Yes, definitely. Whenever we are printing compile errors and compile warnings they should always go to the same place.

Reply:
Ok. I think I misread the issue's original comment regarding "glob miss warnings" to include "compiler warnings" since the "compiler errors" row didn't say "compiler errors and compiler warnings"
This has been fixed in latest commit.